runtime/doc/lua.txt

```diff
@@ -622,6 +622,15 @@ Node methods						*lua-treesitter-node*
 tsnode:parent()						*tsnode:parent()*
 	Get the node's immediate parent.
 
+tsnode:iter_children()					*tsnode:iter_children()*
+	Iterates over all the direct children of {tsnode}, regardless of
+	wether they are named or not.
```

Comment:
I know this was merged but small typo: "wether" => "whether"

Reply:
Oops, this will be fixed in a later pr, thanks for reporting!